app/javascript/components/pages/SearchNew.js

```diff
@@ -68,322 +69,338 @@ class SearchNew extends React.Component {
+                      <FormGroup>
+                        <Row>
+                          <Col>
+                            <Label>Internet Access</Label>
+                          </Col>
+                          <Col>
+                            <Input
+                              type="number"
+                              name="internetaccess"
+                              min="0"
+                              max="10"
+                              onChange={this.handleChange}
+                              value={this.state.form.internetaccess}
+                            />
+                          </Col>
+                        </Row>
+                      </FormGroup>
+                      <FormGroup>
+                        <Row>
+                          <Col>
+                            <Label>Leisure & Culture</Label>
+                          </Col>
+                          <Col>
+                            <Input
+                              type="number"
+                              name="leisure"
+                              min="0"
+                              max="10"
+                              onChange={this.handleChange}
+                              value={this.state.form.leisure}
+                            />
+                          </Col>
+                        </Row>
+                      </FormGroup>
+                      <FormGroup>
+                        <Row>
+                          <Col>
+                            <Label>Tolerance</Label>
+                          </Col>
+                          <Col>
+                            <Input
+                              type="number"
+                              name="tolerance"
+                              min="0"
+                              max="10"
+                              onChange={this.handleChange}
+                              value={this.state.form.tolerance}
+                            />
+                          </Col>
+                        </Row>
+                      </FormGroup>
+                      <FormGroup>
+                        <Row>
+                          <Col>
+                            <Label>Outdoors</Label>
+                          </Col>
+                          <Col>
+                            <Input
+                              type="number"
+                              name="outdoors"
+                              min="0"
+                              max="10"
+                              onChange={this.handleChange}
+                              value={this.state.form.outdoors}
+                            />
+                          </Col>
+                        </Row>
+                      </FormGroup>
+                    </Col>
+                  </Row>
+                </Form>
+              </div>
+            </div>
           </div>
-        </div>
-        {/* {this.state.success && (
+          {/* {this.state.success && (
```

Comment:
optional: remove comments. It may be possible to remove the <> Fragment if there is only on child of <> remaining.

Reply:
Taken care of